temp#1.py

```diff
@@ -0,0 +1,131 @@
+# Calista Adler and Brandon Zhao
+
+import pandas as pd
+import csv 
+
+# opening and reading data from records.csv file 
+data = pd.read_csv(r'/Users/calistaadler/Downloads/records.csv')
+
+# writing into new csv file 
+f = open('/Users/calistaadler/Downloads/tempout.csv', 'w')
+writer = csv.writer(f)
```

Comment:
This is one way to write to a CSV, but a better way is to directly save a dataframe. For example, you can make an output dataframe with 
`output  = pd.DataFrame(columns=['Timestamp', 'ModA Temp', 'ModB Temp', 'ModC Temp', 'Gate Driver Board Temp'])`
then once you've added enough rows to it, save it with `output.to_csv('<filename'>)`

Reply:
Can we still use `writer = csv.writer(output)` using this method?

Reply:
You can, but there is really no reason to and I don't think you should. the to_csv method of a data frame should take care of all writing to the file.